src/DynamicForm.vue

```diff
@@ -74,6 +75,33 @@ import {computed, defineComponent, onMounted, PropType, watch} from "vue";
         emits: ["validate", "update:formMeta", "submit", "confirm"],
 
         setup(props, {emit}) {
+            const options = {
+                // Set float distance so that the arrow does not overlap
+                // the tooltip icon. This was causing multiple mouse
+                // events to be triggered causing the tooltip to flicker
+                distance: 12,
+                themes: {
+                    'tooltip': {
+                        html: true,
+                        delay: {
+                            show: 0,
+                            hide: 0,
+                        },
+                    },
+                }
+            };
+            // we hijack the install function and give it a dummy app
+            // so it doesn't error when it executes app.directive or
+            // app.component but we want it to deep assign the options
+            // to floating vue's internal config object
+
+            // This is a way to configure global options of a directive
+            // in a component, i.e. local version of app.use(plugin, options)
+            const dummyApp = {
+                directive: function (name: string, component: any) {},
+                component: function (name: string, component: any) {}
+            };
+            install(dummyApp, options);
```

Comment:
It did work without adding this line, why is this needed?

Reply:
We can do same in the plugin here https://github.com/reside-ic/vue-next-dynamic-form/blob/482517e0e62bf2d236fcc5d37d6bde83e66548ec/src/plugin.js#L4

Reply:
yep that has definitely simplified the logic massively! thanks :slightly_smiling_face: